app/src/main/java/com/picone/go4lunch/presentation/ui/fragment/RestaurantListFragment.java

```diff
@@ -41,9 +45,12 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         return mBinding.getRoot();
     }
 
+    @SuppressWarnings("ConstantConditions")
+    //mAuth.getCurrentUser().getEmail() can't be null cause needed to enter the app
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
```

Comment:
Why are you querying the User as it's already known to LoginViewModel and UserViewModel

Reply:
Login View model doesn't know the current user, it's just an authentication state view model, and in the UserViewModel i set current user with the value of a new user to add it in db, but i never request the user in db. If i don't do this request, i never get access to user daily schedule. Perhaps there is a way to change this and do more simple.